deploy/scripts/src/cri-kiwi/ipvr.ts

```diff
@@ -152,31 +115,24 @@ const awsConfig = new AWSConfig({
 const sqs = new SQSClient(awsConfig)
 
 export function authEvent (): void {
-  const payload = generateAuthRequest()
+  const authPayload = generateAuthRequest()
 
-  const authEventMessage = {
-    messageBody: JSON.stringify(payload)
-  }
+  const authEventMessage = JSON.stringify(authPayload)
 
-  sqs.sendMessage(env.sqs_queue, authEventMessage.messageBody)
+  sqs.sendMessage(env.sqs_queue, authEventMessage)
 }
 
-export function f2fEvent (): void {
-  const payload = generateF2FRequest()
+export function allEvents (): void {
+  const authPayload = generateAuthRequest()
+  const f2fPayload = generateF2FRequest()
+  const ipvPayload = generateIPVRequest()
 
-  const f2fEventMessage = {
-    messageBody: JSON.stringify(payload)
-  }
-
-  sqs.sendMessage(env.sqs_queue, f2fEventMessage.messageBody)
-}
+  const authEventMessage = JSON.stringify(authPayload)
+  const f2fEventMessage = JSON.stringify(f2fPayload)
 
```

Comment:
Could probably tidy up some of these blank new lines

Reply:
Done